apps/meteor/app/api/server/v1/qrcode.ts

```diff
@@ -0,0 +1,27 @@
+
+
+
+import QRCode from 'qrcode';
+import crypto from 'crypto';
+import { generateJWT } from '/app/utils/server/lib/JWTHelper';
+import { API } from '../api';
+//Doesnt work here
```

Comment:
for this route to work (to be registered) you need to import this file, I recommend doing it here:
https://github.com/RocketChat/Rocket.Chat/blob/960a9149dda3d853b9494ac13b95ccdbcde0c60e/apps/meteor/app/api/server/index.ts#L51